src/Majora/Framework/Loader/Bridge/Form/Type/EntityCollectionType.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Majora\Framework\Loader\Bridge\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Majora\Framework\Loader\LoaderCollection;
+use Majora\Framework\Loader\Bridge\Form\DataTransformer\EntityCollectionTransformer;
+
+/**
+ * Class EntityCollectionType
+ */
+class EntityCollectionType extends AbstractType
```

Comment:
I've removed the DataTransformer as it's not required (i've made a mistake) because the `ChoiceToValueTransformer` already do the job !